docs/concepts/overview/kubernetes-api.md

```diff
@@ -24,13 +24,13 @@ In our experience, any system that is successful needs to grow and change as new
 
 What constitutes a compatible change and how to change the API are detailed by the [API change document](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel/api_changes.md).
 
-## OpenAPI and Swagger definitions
+## OpenAPI definition
 
-Complete API details are documented using [Swagger v1.2](http://swagger.io/) and [OpenAPI](https://www.openapis.org/). The Kubernetes apiserver (aka "master") exposes an API that can be used to retrieve the Swagger v1.2 Kubernetes API spec located at `/swaggerapi`. You can also enable a UI to browse the API documentation at `/swagger-ui` by passing the `--enable-swagger-ui=true` flag to apiserver.
```

Comment:
Let's not remove where people can find swagger specs. You can remove the next paragraph, but this and the one that goes like `Starting with kubernetes 1.4...`  are still valuable.